# DHI packaging: hermetic-build shims for subversion on trixie.
#
# Use file-rename (trixie rename package ships /usr/bin/file-rename); allow
# root builds in the DHI builder; list unpackaged tools in not-installed.
#
# Upstream packaging: https://salsa.debian.org/jamessan/subversion/-/tree/debian/1.14.5-3
--- a/debian/rules	2025-04-06 16:59:00.000000000 +0000
+++ b/debian/rules	2026-08-02 04:22:51.908465478 +0000
@@ -264,7 +264,6 @@
 	  $(MAKE_B) $@ SWIG_PY_PYTHON=python$$v PYTHON=python$$v PYVER=$$v $(check_defs))
 
 override_dh_auto_build-arch:
-	$(DONT_BE_ROOT)
 	# note: autogen-swig does not support -jN
 ifdef DEB_OPT_WITH_PYTHON
 	$(call allpy, $(MAKE_B) -j1 autogen-swig-py)
@@ -375,7 +374,7 @@
 			   mailer/mailer.py verify-po.py svnperms.py;
 
 	# Remove suffixes from binaries in /usr/bin
-	rename 's/\.(sh|pl|rb|py)$$//' debian/*/usr/bin/*
+	file-rename 's/\.(sh|pl|rb|py)$$//' debian/*/usr/bin/*
 
 	pod2man -c 'User Commands' -r "" debian/contrib/svn-clean \
 		$(DEB_BUILDDIR)/svn-clean.1
--- a/debian/not-installed	2025-04-06 16:59:00.000000000 +0000
+++ b/debian/not-installed	2026-08-02 04:22:51.908465478 +0000
@@ -1,2 +1,7 @@
 usr/lib/*/*.la
 usr/lib/*/*.a
+
+usr/bin/svnconflict
+usr/bin/svnmover
+usr/bin/x509-parser
+usr/lib/apache2/modules/mod_dontdothat.so
